### DISCUSSION TITLE
Logos of Institutions/Financing Programs

### DISCUSSION THREAD

**Discussion by Marcelo-Alves7** [joaomlourenco/novathesis#410] [Q&A] [answered]
Hello,

I would like to ask for help understanding the most practical way to include the logos of the institutions/programs that participate in financing my scholarship.
I am an FCT scholarship holder and as far as I know, it is mandatory to include the logos of the FCT and the Portuguese Republic/Science, Technology and Higher Education (Article 30 of the RBI).
The two logos can be found at the following address: https://www.fct.pt/media/media-e-identidade-de-marca/
Maybe add a page (with logos and references) after the acknowledgments and before the abstract?

Thanks in advance for the help.

**Comment by joaomlourenco** [comment]
@Marcelo-Alves7,
Excellent question. This is a point people tend to ignore and shouldn't.
I suggest to include the logos in the back cover, in the bottom, from left to right.  I'll do an example and add it here.
By the way, which version of the template are you using?

**Reply by joaomlourenco** [reply]
This is my suggestion… will add it to the latest version!  If you have troubles updating to the latest version let me know and I'll help you deal with it!
![image](https://user-images.githubusercontent.com/2064643/218277534-1429bb49-054b-4402-b90c-b94722ad1fd4.png)

**Comment by joaomlourenco** [comment, marked as answer]
Done.  Available in Version 6.10.18.
In `Conig/3_covers.tex` search for `\ntsponsor` and uncomment the two “logo” commands for the proper language.

**Reply by b-pereira** [reply]
@joaomlourenco maybe you should includ this on NOVAThesis' UMINHO wiki page. People might still ask about it and how to do it. So, an updated reference could avoid people asking the same question.

**Reply by joaomlourenco** [reply]
@b-pereira The `\sponsor` command only works for FCT-NOVA (with logos) and (hopefully) FC-ULISBOA (with text).  Where should sponsors' logos be included in UMINHO covers?

**Reply by b-pereira** [reply]
As @Marcelo-Alves7 mentioned, and I mentioned to you in past conversations, the elements can be logos or text, but the cover text must be the reverse of the cover color, e.g.: MSc has a white cover and Pantone Cool Gray font; PhD has a Pantone Cool Gray cover and white font color. Good taste suggests putting those elements aligned to the bottom at the same distance from the edge as the front page bottom date (10mm) and with a 25mm margin (left and right).

**Comment by Marcelo-Alves7** [comment]
Professor, that sounds like a great suggestion.
Updated version [6.10.18], but I state that only the funding sponsors for the FCT-NOVA template and FC-ULISBOA were updated.

As I was able to verify in Uminho's formatting rules, and I quote: "On the back covers, there is no obligation to include any information, however, they may be used for the introduction of other elements such as logos or names of entities that supported or financed the work (eg FCT, Erasmus, etc.). These indications should, whenever possible, appear in a single color (white, in the case of a gray background; gray, in the case of a white background) and at the bottom of the page ."
Will it be updated to the UMinho defaults?

Thanks professor.

**Reply by joaomlourenco** [reply]
I will gladly support the UMinho community, which I love for all the enthusiasm and care I have received back! :)
But It is unclear for me how to combine logos and text.  Have to think about it.
But the final answer is "yes!", it will be added to the UMinho variant of the template.  :)

**Reply by Marcelo-Alves7** [reply]
Thank you professor :)

**Reply by b-pereira** [reply]
@joaomlourenco you don't. Or you use the logos (along with the text that is part of the logo as they are), or you include _just text_. The way you put the logos here **is enough.** E.g. if they want to put another financing grant, they can put it in the names as the text.

> This is my suggestion… will add it to the latest version! If you have trouble updating to the latest version let me know and I'll help you deal with it! ![image](https://user-images.githubusercontent.com/2064643/218277534-1429bb49-054b-4402-b90c-b94722ad1fd4.png)

**Reply by Marcelo-Alves7** [reply]
Hello professor,
Regarding this situation, did the professor achieve find an optimal way to include financing logos in the template UMinho?
I think it could be something similar to the founding sponsor for the FCT-Nova Template that you generated above, just appearing these 2 logos (FCT and Portuguese Republic) on the back cover and at the bottom of the page. 

Sorry to open this discussion again, but I need your help to count on with this variant in the template UMinho, since I'm near to deliver the thesis.
Thank you very much.

**Reply by joaomlourenco** [reply]
Working on double checking the UMinho in v7.0.0. More news later today.

**Comment by joaomlourenco** [comment]
Just pushed some corrections to UMINHO in branch 7.0.0.
Also added support for sponsor logos in the back cover.
Can you please try it?

**Comment by joaomlourenco** [comment]
@b-pereira 
May I ask you to please check the branch 7.0.0??  Thank you.

_Just pushed some corrections to UMINHO in branch 7.0.0.
Also added support for sponsor logos in the back cover.
Can you please try it?_

**Reply by b-pereira** [reply]
I am busy right now. I will try it soon.
